### DISCUSSION TITLE
Encapsulation file migration

### DISCUSSION THREAD

**Discussion by ChaiWenyu1983** [filecoin-project/lotus#9388] [Storage Provider] [answered]
Copy the package file to the new storage directory. The old directory is going to be invalidated. The copy is always in the "error" status. How to solve this problem?

Use the "lotus miner storage find" command to view that the file is in the new directory.

**Comment by rjan90** [comment, marked as answer]
If you want to move sector files from on storage path to another storage path, you will either have to redeclare them following this guide for updating their locations: https://lotus.filecoin.io/storage-providers/operate/custom-storage-layout/#updating-locations.

Else you can restart the `lotus-miner` and it should automatically redeclare in which storage path the sectors are in 😄

**Comment by ChaiWenyu1983** [comment]
For example, can I manually execute "lotus miner storage redemption -- id 365" after the COPY "s-t0186 ****** - 365" file is transferred to the new storage directory?






&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "filecoin-project/lotus"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月28日(星期三) 下午2:07
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [filecoin-project/lotus] Encapsulation file migration (Discussion #9388)





 
If you want to move sector files from on storage path to another storage path, you will either have to redeclare them following this guide for updating their locations: https://lotus.filecoin.io/storage-providers/operate/custom-storage-layout/#updating-locations.
 
Else you can restart the lotus-miner and it should automatically redeclare in which storage path the sectors are in 😄
